integreat_cms/cms/constants/roles.py

```diff
@@ -116,6 +117,7 @@
     "change_poi",
     "change_pushnotification",
     "change_region",
+    "change_chatmessage",
```

Comment:
I would argue that the marketing team should also be allowed to use the chat

Reply:
@JoeyStk If you resolve conversations without making changes, please write a short comment why you think it's irrelevant :slightly_smiling_face: